README.md

```diff
@@ -394,6 +394,20 @@ in the Installation Guide.
     or JupyterLab installation, as described in [section
     "Launching SageMath"](https://doc.sagemath.org/html/en/installation/launching.html)
     in the installation manual.
+    
+Alternative Installation using PyPi
+---------------
+
+For installation of `sage` in python using `pip` you need to install `sagemath-standard`. First, activate your python virtual environment and follow these steps:
+
+            $ python3 -m pip install sage_conf
+            $ ls $(sage-config SAGE_SPKG_WHEELS)
+            $ python3 -m pip install $(sage-config SAGE_SPKG_WHEELS)/*.whl 
+            $ python3 -m pip install sagemath-standard
+
+You need to install the `sage_conf`, a wheelhouse of various python packages. You can list the wheels using `ls $(sage-config SAGE_SPKG_WHEELS)`. After manual installation of these wheels, you can install the sage library, `sagemath-standard`. 
+
+**NOTE:** You can find `sage` and `sagemath` pip packages but it is worth noting that they are not at all related to this `sage`. Indeed `pip install sage` installs `sage 0.0.0` which is literally an empty package. 
```

Comment:
After installing `sage` and `sagemath` with `pip` and importing `sage` it throws an import error: `No module named sage`. 
I don't understand how they are related to sage if they can't import sage at all. Please refer to [this](https://stackoverflow.com/questions/50039916/no-module-named-sage-all) post on stackoverflow.

Reply:
There's nothing deep to understand here. Both packages were created because the respective authors intended to provide some functionality related to SageMath. But neither of the two packages can be used to install SageMath, so they should be ignored.

Reply:
Ok, I will update the readme